pkg/ddc/alluxio/ufs.go

```diff
@@ -90,6 +93,9 @@ func (e *AlluxioEngine) ShouldUpdateUFS() (ufsToUpdate *utils.UFSToUpdate) {
 	ufsToUpdate = utils.NewUFSToUpdate(dataset)
 	ufsToUpdate.AnalyzePathsDelta()
 
+	// 3. for hostpath ufs mount, check if all mountpoints have been mounted
+	e.remountOnMasterRestart(ufsToUpdate)
```

Comment:
The fix looks pretty good. Thanks. Only one suggestion. How about changing the method name from `remountOnMasterRestart` to `checkIfRemountRequired`?

Reply:
Good suggestion!